src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -57,7 +57,8 @@
     "conflict": {
         "phpdocumentor/reflection-docblock": "<3.0",
         "phpdocumentor/type-resolver": "<0.2.0",
-        "symfony/console": "<3.3"
+        "symfony/console": "<3.3",
+        "symfony/web-profiler-bundle": "<3.3"
```

Comment:
`"symfony/web-profiler-bundle": "~3.3"` must be added as dev requirement in addition of the conflict, that should make the build green.